frontend/src/app/app.module.ts

```diff
@@ -37,7 +39,11 @@ import { SideMenuComponent } from './side-menu/side-menu.component';
 
 // Imports for Plotly.
 import { PlotlyModule } from 'angular-plotly.js'
-import * as PlotlyJS from 'plotly.js/dist/plotly.js'
+import * as PlotlyJS from 'plotly.js/dist/plotly.js';
```

Comment:
Does order matter here? If not, should be alphabetical, and the * should be first or last.